packages/expo-cli/src/commands/eas-build/build/utils/configureUpdatesAsync.ts

```diff
@@ -0,0 +1,276 @@
+import { Platform } from '@expo/build-tools';
+import { getConfig, getExpoSDKVersion, getPackageJson } from '@expo/config';
+import { UserManager } from '@expo/xdl';
+import chalk from 'chalk';
+import dedent from 'dedent';
+import figures from 'figures';
+import * as fs from 'fs-extra';
+import glob from 'glob';
+import { reject } from 'lodash';
+import ora from 'ora';
+import path from 'path';
+import xcode from 'xcode';
+import { DOMParser, XMLSerializer } from 'xmldom';
+
+import { gitAddAsync } from '../../../../git';
+import log from '../../../../log';
+import * as gitUtils from '../../utils/git';
+
+interface UpdateOptions {
+  sdkVersion: string;
+  runtimeVersion?: string;
+  updateUrl: string;
+}
+
+const platformDisplayNames = {
+  [Platform.Android]: 'Android',
+  [Platform.iOS]: 'iOS',
+};
+
+export default async function configureUpdatesAsync({
+  projectDir,
+  nonInteractive,
+  platform,
+}: {
+  projectDir: string;
+  nonInteractive: boolean;
+  platform: Platform;
+}) {
+  const user = await UserManager.ensureLoggedInAsync();
+
+  const packageJson = getPackageJson(projectDir);
+
+  if (!(packageJson.dependencies && packageJson.dependencies['expo-updates'])) {
+    return;
+  }
+
+  const spinner = ora(`Configuring expo-updates for ${platformDisplayNames[platform]}`);
+
+  const { exp } = getConfig(projectDir);
+  const sdkVersion = getExpoSDKVersion(projectDir, exp);
```

Comment:
What if there is no sdkVersion? Does expo-updates support that case?

Reply:
@wkozyra95 expo-updates requires either sdk version or runtime version. `getExpoSDKVersion` is always gurranteed to return sdk version so it should be ok.

Reply:
getExpoSDKVersion is throwing exception if there is no sdkVersion in app.sjon no `expo` package installed

Reply:
do you think I should only try to get sdk version if no runtime version is present?

Reply:
yes
runtimeVersion should take precedence
but if neither of those values is present we should throw some better error than what `getExpoSDKVersion` throws